apps/web/src/pages/CommunityInterests/sections/ReviewAndSubmit.tsx

```diff
@@ -0,0 +1,164 @@
+          {intl.formatMessage({
+            defaultMessage: "Review and submit",
+            id: "ITtt88",
+            description: "Heading for the 'Review and submit' section",
+          })}
+        </Heading>
+        <span>
+          {intl.formatMessage({
+            defaultMessage:
+              "Please review the information you’ve provided. Once a community is added to your profile, you can freely update this information or remove the community entirely at any time.",
+            id: "o9e3P/",
+            description: "Description of the 'Review and submit' section",
+          })}
+        </span>
+      </div>
+      <CardSeparator space="none" />
+      {/* consent form */}
+      <div
+        data-h2-display="base(flex)"
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1)"
+      >
+        <Heading
+          level="h3"
+          data-h2-font-weight="base(400)"
+          size="h4"
+          color="primary"
+          data-h2-margin="base(0)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Consent to share your information",
+            id: "HMlreE",
+            description: "Heading for the 'Review and submit' consent section",
+          })}
+        </Heading>
+        {/* consent statement */}
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column)"
+          data-h2-gap="base(x0.5)"
+        >
+          <p>
+            {intl.formatMessage({
+              defaultMessage:
+                "When you express interest in job or training opportunities through a functional community, you’re agreeing to share your profile information with recruiters and potential hiring managers. This includes information shared in your profile, your career experience, and your skills portfolio.",
+              id: "mv2nt/",
+              description:
+                "statement for the 'Review and submit' consent section, paragraph 1",
+            })}
+          </p>
+          <p>
+            {intl.formatMessage({
+              defaultMessage:
+                "You can rescind this consent at any time by opting out of job referrals and interest in training.",
+              id: "ot7s0V",
+              description:
+                "statement for the 'Review and submit' consent section, paragraph 2",
+            })}
+          </p>
+        </div>
+        {formRequiresConsent ? (
+          <Checkbox
+            id="consent"
+            name="consent"
+            boundingBoxLabel={intl.formatMessage({
+              defaultMessage: "Consent to share",
+              id: "AlOrGy",
+              description: "Label for the input for the consent check",
+            })}
+            label={intl.formatMessage({
+              defaultMessage:
+                "I agree that by indicating my interest in work or training opportunities that my profile will be shared with HR staff and hiring managers in this functional community.",
```

Comment:
Avoid abbreviations for accessibility purposes.

```suggestion
                "I agree that by indicating my interest in work or training opportunities that my profile will be shared with human resources staff and hiring managers in this functional community.",
```

Reply:
I'd argue that HR is a well-known enough abbreviation that it doesn't need to be written in full in this context here. similar perhaps to the IT abbreviation.

Reply:
While this may be a widely used English abbreviation the concern is for those that have English as a second language that may not be aware